lib/chess_web/views/game_view.ex

```diff
@@ -1,3 +1,13 @@
 defmodule ChessWeb.GameView do
   use ChessWeb, :view
+
+  import Chess.Auth, only: [current_user: 1]
+
+  def opponent(conn, game) do
```

Comment:
It seems like this function and `get_opponents` may be something to keep an eye for extracting.  I'm not sure if it makes to put them in the same module or not though.